core/storage/base_model/gae_models.py

```diff
@@ -33,6 +33,10 @@
 RAND_RANGE = (1 << 30) - 1
 ID_LENGTH = 12
 
+# Types of deletion policies.
+DELETION_POLICY = utils.create_enum(  # pylint: disable=invalid-name
```

Comment:
Try to avoid using pylint-disable. Why is the name invalid? (The corresponding one in core.platform.models isn't.)

Reply:
I'm not sure why the name is invalid here but not in `core.platform.models`, the error is saying `Class name "DELETION_POLICY" doesn't conform to PascalCase naming style`.

Reply:
Hm, I think you might need to investigate a bit more why it's invalid here -- we shouldn't have an exception. /cc @kevinlee12 

Note that .pylintrc does not show anything related to PascalCase, so I'm not sure where this is coming from...

Reply:
The PascalCase error belongs under [invalid-name](http://pylint-messages.wikidot.com/messages:c0103). Because `DELETION_POLICY` is Enum (which is basically a class) thus it is evaluated by the class regex (PascalCase, `[A-Z_][a-zA-Z0-9]+$`), `MODELS` are evaluated by the same regex but because it is just one word it passes it.

For some reason the usage of UPPER_CASE seems more appropriate in this cases, but we can probably switch to PascalCase for these enums.

Reply:
Hm yeah, I agree UPPER_CASE seems more appropriate. Let's keep the pragma, but add a comment to explain why the pragma is needed. (The point about this being an Enum which is basically a class and is evaluated by pylint as one is the important part here.)